filter_plugins/ovirtvmip.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
```

Comment:
is this file called ovirtvmip? You may want to consider renaming to more relevant name

Reply:
Yes, I will rename this to appropriate name